src/Services/servicesimpl.ts

```diff
@@ -0,0 +1,123 @@
+import schemarepo from "../repository/impl/repositoryimpl";
+import logger from "../Core/logger";
+import dir from "../repository/schemas";
+import { GetResponse } from "../Dto/Response/getRes";
+import { Response } from "../Dto/Response/createRes";
+import { PhoneEntry } from "../Dto/Request/createReq";
+import { UpdateResponse } from "../Dto/Response/updateRes";
+import { UpdateManyPhoneEntry } from "../Dto/Request/updateReq";
+import { manyUpdateResponse } from "../Dto/Response/updateRes";
+import { UpdatePhoneEntry } from "../Dto/Request/updateReq";
+import { IServices } from "./services";
+import { manyEntry } from "../Dto/Request/createReq";
+import { manyResponse } from "../Dto/Response/createRes";
+import { paginationresponse } from "../Dto/Response/getRes";
+
+class Services implements IServices {
```

Comment:
rename to phoneBookService Services is generic also use singular for classname

Reply:
done